spec/rubyspec/core/enumerable/filter_spec.rb

```diff
@@ -0,0 +1,7 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+require File.expand_path('../shared/find_all', __FILE__)
```

Comment:
No require_relative in specs.
If we update it, we should update it for all specs (in https://github.com/ruby/spec/) to remain consistent.

New specs can be directly here though, that's perfectly fine.